### ISSUE TITLE
OUTPUT has an error

### ISSUE THREAD

**Issue by LHyphen** [danielpinto8zz6/c-cpp-compile-run#335]
**Describe the bug**
When Compile, Run & Debug, an error will be prompted in the output
 `Error: there is no registered task type 'Compile'. Did you miss installing an extension that provides a corresponding task provider?`

**To Reproduce**
1. Compile, Run & Debug code
2. Click on 'OUTPUT'
3. See error

**Expected behavior**
no error

**Screenshots**
![image](https://github.com/danielpinto8zz6/c-cpp-compile-run/assets/83161242/45d03a13-1f2a-4dc7-90f0-c8ea31a6bbd6)

**Environment**
 - VSCode Version: 1.81.1
 - OS Version: Windows_NT x64 10.0.19044

**Additional context**

**Comment by danielpinto8zz6** [comment]
Does the code compile?

**Comment by LHyphen** [comment]
can compile

**Comment by danielpinto8zz6** [comment]
So, the error is logged but everything works?

**Comment by LHyphen** [comment]
Yes, although this error is prompted, the functions are normal. 
Two error prompts will appear every time Compile, Run & Debug.
Besides me, I know another person who also has this situation, it should not be an exception.

**Comment by danielpinto8zz6** [comment]
Ok, I'll try to replicate the error

**Comment by niamotullah** [comment]
> **Describe the bug** When Compile, Run & Debug, an error will be prompted in the output `Error: there is no registered task type 'Compile'. Did you miss installing an extension that provides a corresponding task provider?`
> 
> **To Reproduce** Steps to reproduce the behavior:
> 
>     1. Compile, Run & Debug code
> 
>     2. Click on 'OUTPUT'
> 
>     3. See error
> 
> 
> **Expected behavior** no error
> 
> **Screenshots** ![image](https://user-images.githubusercontent.com/83161242/263531832-45d03a13-1f2a-4dc7-90f0-c8ea31a6bbd6.png)
> 
> **Environment**
> 
>     * VSCode Version: 1.81.1
> 
>     * OS Version: Windows_NT x64 10.0.19044
> 
> 
> **Additional context** Add any other context about the problem here.

I'm also facing the same issue.

	Version: 1.83.0
	Commit: e7e037083ff4455cf320e344325dacb480062c3c
	Date: 2023-10-03T16:11:49.843Z
	Electron: 25.8.4
	ElectronBuildId: 24154031
	Chromium: 114.0.5735.289
	Node.js: 18.15.0
	V8: 11.4.183.29-electron.0
	OS: Linux x64 6.2.0-34-generic snap

**Comment by danielpinto8zz6** [comment]
Hey @niamotullah if you install the c/c++ extension does the problem persists?

**Comment by niamotullah** [comment]
> Hey @niamotullah if you install the c/c++ extension does the problem persists?

Yes, After installing Microsofts  _C/C++_ extension problem doesn't persists.

**Comment by RimuruChan** [comment]
> Hey @niamotullah if you install the c/c++ extension does the problem persists?

Yes, it persists though i installed it.

Os: Windows 11
Compiler: msys2 ucrt64 gcc

**Comment by RimuruChan** [comment]
And this seems to also cause compiling slower, because the error takes a few seconds.

**Comment by zhk1211** [comment]
> Hey @niamotullah if you install the c/c++ extension does the problem persists?

So how to deal with it.

**Comment by zhk1211** [comment]
![QQ_1726707975978](https://github.com/user-attachments/assets/cdf55e94-789c-4f02-97a7-7e8b0fe1d02b)

**Comment by danielpinto8zz6** [comment]
It should be fixed by this: https://github.com/danielpinto8zz6/c-cpp-compile-run/pull/371
Can you guys confirm please?
